

#### Es6中的错误监管机制
通过try catch可以对错误进行一个监管，如果在执行时出现了错误，就会执行catch中设置的代码，这样就避免了出错阻塞页面的情况，只能监管同步执行的代码。具体的示范代码如下:
```js
    // 错误监管
    try {
        console.log(a); // 没有a，就会报错
    } catch (e) {       // 避免报错，报错执行内部的函数
        console.log("===");  
    }

    // 不能监管异步中的错误
    try {
        setTimeout( () => {
            // 会直接报错，阻塞页面运行，不会执行catch中的函数
            console.log(a); 
        }, 0)
    } catch (e) {       
        console.log("===");  
    }
    // 如果想要监管异步中的错误，就需要在异步执行的函数中加上try catch
```

